README.md

```diff
@@ -299,6 +299,8 @@ Here is a complete list of configuration directives we support:
   declared dependencies. Must be one of `"requirements.txt"`, `"setup.py"`,
   `"setup.cfg"`, `"pyproject.toml"`, or leave it unset (i.e. the default) for
   auto-detection (based on filename).
+- `install-deps`: Install Python dependencies gathered with FawltyDeps to
+  a temporary virtual environment. Dependencies will be installed from PyPI.
```

Comment:
I'd like this part to be a little more precise (and verbose) about how we install and where packages come from. Specifically, we only install packages from PyPI because that is where `pip install` goes by default. Therefore, I think we're better off mentioning `pip install` specifically here (so that users can infer that `pip`'s configuration/environment variables are what they should look at if default `pip` behavior is not suitable):

```suggestion
- `install-deps`: Automatically install Python dependencies gathered with
  FawltyDeps into a temporary virtual environment. This will use `pip install`,
  which downloads packages from PyPI by default.
```